src/pages/Login/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import './styles.scss'
+import TextInput from './components/TextInput'
+import { setPlayersInfo } from '../../providers/playersInfo'
+import { useHistory } from "react-router-dom";
+
+const TITLE = 'Tic Tac Toe Intergalactic'
+const SUBTITLE = 'Conquer planets with your friends'
+const FORM_TITLE = 'Enter your names to start playing'
+
+const validatePlayerNames = ({ firstPlayer, secondPlayer }) => {
+  return firstPlayer && secondPlayer && firstPlayer !== secondPlayer
+}
+
+const Login = () => {
+  const history = useHistory();
+  const [firstPlayer, setFirstPlayer] = useState('')
+  const [secondPlayer, setSecondPlayer] = useState('')
+  
+  const handleSubmit = () => {
+    if (validatePlayerNames({ firstPlayer, secondPlayer })) {
+      setPlayersInfo({ firstPlayer, secondPlayer });
+      history.push("/game")
+    } else {
+      alert("Debes completar los campos correctamente")
+    }
+  }
+
+  return (
+    <div className='container'>
+      <div className='splashText'>
+        <p className='title'>{TITLE}</p>
+        <p className='subtitle'>{SUBTITLE}</p>
+      </div>
+      {/*mover a componente aparte*/}
```

Comment:
Same `TODO` suggestion here. It would be like: `TODO: Move form block to its own component`